pyhoma/enums.py

```diff
@@ -6,13 +6,15 @@
 
 @unique
 class ProductType(IntEnum):
+    UNKNOWN = -1
```

Comment:
You miss the missing method

Reply:
Oops, I decided to not add the method since this one is pretty stable and shouldn't change much.

But I forgot to remove the unknown enum. Would you agree that adding the extra logic is overkill? (Since we don't do it for all enums).

Reply:
Yes it will be overkill indeed here.